Objects/longobject.c

```diff
@@ -4185,6 +4185,7 @@ long_pow(PyObject *v, PyObject *w, PyObject *x)
                 goto Error;
             Py_DECREF(a);
             a = temp;
+            temp = NULL;
```

Comment:
Urk! That one looks like my fault. We probably want to pull this into a separate PR, so that it can be backported. I'll aim to do that at some point this weekend.

Reply:
Sounds go to me! Thank you. I removed the change from this PR.